docs/static/exampleinclude.css

```diff
@@ -84,3 +84,15 @@
   color: #404040;
   background-color: #d6d6d6;
 }
+
+/* override the maximum width restriction for large screen */
```

Comment:
This doesn't look like code related to the exampleinclude directive.

Reply:
Kindly suggest file name where this code can be placed.

Reply:
I don't think that is a good solution. The lack of text width on all pages makes some pages very difficult to read on some screens. I would prefer to turn these tables into lists to make the whole content narrower. I currently have these tasks in my queue, but I want to do it cleverly and automatically, and then use this information to update the website - https://github.com/apache/airflow-site/blob/master/landing-pages/site/static/integrations.json

> According to several sources designers should strive for 45 to 85 characters per line since the ideal width for a paragraph is supposedly 65 characters.

https://medium.com/@matuzo/writing-css-with-accessibility-in-mind-8514a0007939